### ISSUE TITLE
Shikoku 1889 - add corporations icon to the starting locations on the map

### ISSUE THREAD

**Issue by konradh-cmyk** [tobymao/18xx#11022]
Hello,
sorry if this was already rised, couldn't find similar ticket.

I know there is marking done by letters like "IR" or "UR", but I think it would greatly improve experience of new players if the corporation logos were included on the map. Since game is already using the icons in other parts of the interface, maybe this shouldnt be a problem?

Best

**Comment by philcampeau** [comment]
As described, I don't know that this will be implemented. 

That said, we could probably write some code that could display a faded gray-scale version of the logos on the home city spaces, sort of like how destination stations are shown on the physical map for 1846.

If we do that, I think it would also be nice to add some code that would also display the home stations on laid tiles, since there's no reason we can't display that info.

**Comment by arseniybanayev** [comment]
Agree with OP. The colored company icons are used in the Market so they should also be used in the Map. Why is it a different iconography?

**Comment by philcampeau** [comment]
> New user of site here - and also an engineer with long experience so would be very happy to help contribute to this great repo!
> 
> Agree with OP. The colored company icons are used in the Market so they should also be used in the Map. Why is it a different iconography?

If we used the colored icon, would you differentiate between a tokened city and a reserve spot?

**Comment by arseniybanayev** [comment]
Good point. It would have to be a differentiation in value (or shading or something like you recommended). Happy to take a stab at your suggestion and get familiar with contribution guidelines, if that's okay!

**Comment by philcampeau** [comment]
Be my guest! I wasn't interested in taking on this task, just wanted to offer thoughts :)

**Comment by philcampeau** [comment]
Thinking back on this issue, I don't think using a faded grey-scale version of the logo makes sense, precisely _because_ some games use that to denote teleport spaces. 

For now, I'm going to close this request. If anyone wants to implement something like this, they can, but as it stands there's no intention of implementing it.
